Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/LookModificationTransform.azsl

```diff
@@ -21,7 +22,7 @@ ShaderResourceGroup PassSrg : SRG_PerPass_WithFallback
     {
         MinFilter = Linear;
         MagFilter = Linear;
-        MipFilter = Linear;
+        MipFilter = Point;
```

Comment:
Should we change the name of this sampler?

Reply:
Honestly I'll just switch it back to linear for now - every place we're sampling from we use SampleLevel and explicitly request the 0 mip so it doesn't really matter.